pkg/registry/apis/dashboard/search.go

```diff
@@ -303,6 +303,20 @@ func (s *SearchHandler) DoSearch(w http.ResponseWriter, r *http.Request) {
 		}}
 	}
 
+	// The ids filter
+	ids, ok := queryParams["uid"]
+	if ok {
+		if searchRequest.Options.Fields == nil {
+			searchRequest.Options.Fields = []*resource.Requirement{}
+		}
+		idFilter := []*resource.Requirement{{
+			Key:      "_id",
+			Operator: "=",
+			Values:   ids,
```

Comment:
```suggestion
			Key:      "name",
			Operator: "in",
			Values:   ids,
```
I expected this would be an "IN" query... that is essentially a giant OR.  But looks like you have speical handling for _id that is also a disjoint query.

Reply:
yes i used `_id` for this special case...and since we don't index `name`

Reply:
`_id` is the field name of the id field in bleve.  we could maybe do a wildcard query on that field but i assume`query.NewDocIDQuery(values)` is faster/more efficient.